sys/shell/commands/shell_commands_cortexm.ld

```diff
@@ -0,0 +1,9 @@
+SECTIONS
+{
+  .data :
+  {
+    _shell_command_list = .; /* begining of the command list */
+    KEEP(*(._shell_command_list))
+    KEEP(*(._shell_command_list_end))
+  } > ram
```

Comment:
Why is it necessary to put it in RAM in cortex-m?

Reply:
Hello, I don't remember exactly why, but it was something about the main cortexm linker script. Basically, I have tested the code on native and cortexm(3?) targets. Native target puts everything in RAM, so that it is not needed to specify it, while it would make sense to do so for microcontrollers in general, where we need to choose between RAM, EEPROM, Flash.